en/physics-2d/physics-2d-system.md

```diff
@@ -0,0 +1,158 @@
+# Physics Manager
+
+The physics system hides most of the implementation details of the physical modules (Box2D and Builtin modules) (e.g. creating rigid bodies, synchronizing rigid body information to nodes, etc.).
+
+Use the physics system to access some of the common functions of the physics module, such as click testing, ray testing, and setting up test messages.
+
+## Physics system related settings
+
+### Enabling the Physics Manager
+
+The __Physics Manager__ is disabled by default. To use the physics system-related functions, first, enable the __Physics Manager__, otherwise, all the physics objects setup in the editor will not produce any effect.
+
+```ts
+PhysicsSystem2D.instance.enable = true;
+```
+
+### Draw physics debugging information
+
+To enable draw debugging information, use the __debugDrawFlags__.
+
+The physics system provides a variety of debugging information, you can combine the information to draw the relevant content.
+
+```ts
+PhysicsSystem2D.instance.debugDrawFlags = EPhysics2DDrawFlags.Aabb |
+    EPhysics2DDrawFlags.Pair |
+    EPhysics2DDrawFlags.CenterOfMass |
+    EPhysics2DDrawFlags.Joint |
+    EPhysics2DDrawFlags.Shape;
+```
+
+Set the drawing flag to `EPhysics2DDrawFlags.None` to disable drawing.
+
+```ts
+PhysicsSystem2D.instance.debugDrawFlags = EPhysics2DDrawFlags.None;
+```
+
+### Conversion the physics units to the world coordinate system units
+
+General Physical Modules(Box2D) uses __Metre-Kilogramme-Second (MKS)__ unit system, it has the best performance operating under such a unit system. But we use the __world coordinate system units__ (short for world units) as the unit of length in 2D games, so we need a ratio to convert the physics units to the world units.
+
+In general, we set this ratio to __32__, which can be obtained by `cc.PhysicsManager.PTM_RATIO`, and this value is read-only. Usually the user does not need to care about this value, the physics system will automatically convert the physics units and world units to each other. User can use the familiar world units for all the calculations.
+
+### Set physics gravity
+
+Gravity is a very important thing in physics operation, and most physics games use the gravity as a important feature.
+
+The default gravity is `(0, -320)` world units per second^2, according to the conversion rules described above, that's `(0, -10)` m/s^2 in physics unit.
+
+If you want the gravity to be 0, set like this:
+
+```ts
+PhysicsSystem2D.instance.gravity = cc.v2();
+```
+
+If you wish to change the acceleration of gravity to something else, such as a `20 m/s` acceleration landing, then you can set the acceleration to.
```

Comment:
If you wish to change the acceleration of gravity to something else, such as a `20 m/s` acceleration landing, then you can set the acceleration to.
-->
It is possible to change the acceleration of gravity to something else, such as a `20 m/s`. Example: